internal/go-symlink-logfiles-watcher/Makefile

```diff
@@ -0,0 +1,9 @@
+all: build
```

Comment:
Don't need a separate makefile, the Dockerfile (whichever one ends up building this) can just use `go build` and do file copies as needed.